democracy/static/admin/hide-plugin-fields.js

```diff
@@ -0,0 +1,18 @@
+(function($) {
+    $(function() {
+        function setPluginFieldsVisibility($typeSelect) {
+            var $pluginFields = $typeSelect.closest("fieldset").children().has("*[id *=plugin]");
+
+            // hide plugin fields if type select's value is "introduction"
+            $typeSelect.val() === "1" ? $pluginFields.hide() : $pluginFields.show();
```

Comment:
Don't use ternary expressions instead of `if` statements if you're not assigning the retval.

Also, `$pluginFields.toggle($typeSelect.val() === "1");`